### ISSUE TITLE
TX (and others?): Missing committee_id in legislator roles

### ISSUE THREAD

**Issue by grgcombs** [openstates/openstates-scrapers#83]
In some states, when referring to committee positions, legislator roles have the corresponding committee_id included along with the committee name.  However, this doesn't appear to be the case in Texas.  I'm still looking to see if it's an issue elsewhere.

Is this difficult to standardize?  For StatesLege, I would like to use this ID as a very direct route to drill down to a committee's details and its membership from a legislator's list of committee positions (roles)...

Basically, I'd like the capability to treat legislator "roles" as a rough inverse of the committee "members" (where "leg_id" is always present).  I another option would be to break the legislator's committee positions out of into their own array, since "roles" isn't exclusively committee positions.
